app/components/controls/layerControl/LayerControl.tsx

```diff
@@ -44,11 +46,14 @@ export default function LayerControl({
       overlayVisibility.observations && zoomLevel < FETCH_ZOOM_LEVEL_THRESHOLD;
 
     if (sightingsError && observationsError) {
-      setFetchingError("Zoomaa lähemmäksi tietojen hakemista varten");
+      const error = t("sightingsObservationsError");
```

Comment:
You're using `t` here, so should it be part of the `useEffect` dependencies?

Reply:
added to dependencies, good catch.